docs/book/src/user/quick-start.md

```diff
@@ -180,6 +180,23 @@ before getting started with Cluster API. See below for the expected settings for
 
 Download the latest binary of `clusterawsadm` from the [AWS provider releases] and make sure to place it in your path.
 
+```bash
+# Download the specific provider release from the release pages (download example given below will vary based on your choice)
+wget <your system specific release>
+
+# Rename the downloaded binary if required
+mv <downloaded binary> clusterawsadm
+
+# Make it executable
+chmod +x clusterawsadm
+
+# Move the binary to a directory present in your PATH
+sudo mv clusterawsadm /usr/local/bin
+
+# Check version to confirm installation
+clusterawsadm version
+```
```

Comment:
Could we generate a releaselink like we do for clusterctl? 
```
curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-darwin-arm64" version:"1.0.x"}} -o clusterctl
```

These bits are mostly useful for linux users, should we make that clear?

Reply:
Out of curiosity, why Linux users?

Reply:
+1 to make this example something the user can copy and execute on its machine